firebase_admin/_auth_providers.py

```diff
@@ -149,6 +165,15 @@ def __init__(self, http_client, project_id, tenant_id=None):
         if tenant_id:
             self.base_url += '/tenants/{0}'.format(tenant_id)
 
+    def get_oidc_provider_config(self, provider_id):
+        _validate_oidc_provider_id(provider_id)
+        body = self._make_request('get', '/oauthIdpConfigs/{0}'.format(provider_id))
+        return OIDCProviderConfig(body)
+
+    def delete_oidc_provider_config(self, provider_id):
+        _validate_oidc_provider_id(provider_id)
+        self._make_request('delete', '/oauthIdpConfigs/{0}'.format(provider_id))
```

Comment:
Should we return the status 'success/failed' here?

Reply:
Return of these delete methods are `void`.